test/experiment_tests/clock.jl

```diff
@@ -1,5 +1,7 @@
 using Test
```

Comment:
would it be possible to move the `clock.jl` and `coupler_state.jl` tests to one file that corresponds to `experiments/ClimaCore/CoupledSimultations/coupled_simulation.jl`, so it's more contained? We could also retain the `Sim` syntax for the filenemes too.

Reply:
As it is right now, `test/experiment_tests/coupler_state.jl` tests `experiments/ClimaCore/CouplerState/coupler_state.jl`, and `test/experiment_tests/clock.jl` tests `experiments/ClimaCore/CoupledSimulations/coupled_simulation.jl` (which may be confusing naming). Do you think I should still combine the test files, or should I rename the clock one?

Reply:
Renaming is definitely a good idea. I would prefer containing this in as few files as possible, since it's not code we want to retain long term, but either works in practice. :)

Reply:
Sounds good, I agree! I condensed all of the code into `experiments/ClimaCore/CoupledSims/coupled_sim.jl`, and the tests into `test/experiment_tests/coupled_sims.jl`